ducktape/command_line/main.py

```diff
@@ -147,6 +147,10 @@ def main():
             print "    " + str(test)
         sys.exit(0)
 
+    if len(tests) == 0:
```

Comment:
Maybe we should see this message even if --collect-only is flagged? (I.e. move this above the `if args.collect_only` block)

Also, "No test classes found" -> "No tests found"